src/pip/_vendor/tenacity/__init__.py

```diff
@@ -22,10 +22,9 @@
 except ImportError:
     iscoroutinefunction = None
 
-try:
-    import tornado
-except ImportError:
-    tornado = None
+# Pip does not currently vendor Tornado, so don't import it even if it
+# happens to be installed in the environment.
```

Comment:
If I read this comment for the first time, I’d think to myself “OK but that works without the patch as well, so is this unnecessary”?

I’d mention why the patch is *needed* instead to avoid this confusion.

Reply:
Done.